Solutions/CohesitySecurity/Playbooks/Cohesity_Restore_From_Last_Snapshot/azuredeploy.json

```diff
@@ -0,0 +1,358 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
```

Comment:
Kindly add metadata here

Reply:
Done.

Reply:
See the line #7 `"prerequisites": "Deploy the Cohesity configuration and function apps (see details at https://github.com/cohesity/Azure-Sentinel/blob/CohesitySecurity.internal/Solutions/CohesitySecurity/Playbooks/Cohesity_Restore_From_Last_Snapshot/readme.md)",`

Reply:
Yeah I can see now, But doubt that how the provided link will show in solution.

Beacuse link always provided in below format : 
"[click here](https://www.randomurl.com)"